config.example.json

```diff
@@ -2,6 +2,11 @@
   "source": {
     "access_key_id": "",
     "secret_access_key": "",
-    "bucket": ""
+    "bucket": "",
+    "options": [
+      "--dryrun",
+      "--exclude '*'",
+      "--include 'results/*'"
+    ]
```

Comment:
Hmm, maybe we should leave these options out of this file so that they don't accidentally get left in by someone following the instructions to the letter?